src/server/middleware/nation-importer.js

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Wow this is a nice bit of code.  We actually have a NationBuilder API key now, though, so we should be able to ditch the google spreadsheet middleman entirely and have signups go straight to Nationbuilder!